fmings/FEBourbonCollective
# updated sign in page & favicon

## Description
- updated background color for app
- add logo on sign in page
- updated favicon
- confirm sign in with google auth works
## Related Issue
#7 

## Motivation and Context
This change allows users to sign in with google auth and allows for branding upon opening app

## How Can This Be Tested?
Open app
Confirm logo is on sign in page
Click sign in
Confirm google auth sign in works

## Screenshots (if appropriate):

## Types of changes
